README.md

```diff
@@ -1,5 +1,18 @@
 Takeaway Challenge
```

Comment:
you could update your readme to explain how your code works, any challenges you faced, how to run etc. 😊